src/pages/[platform]/build-a-backend/add-aws-services/deletion-backup-resources/index.mdx

```diff
@@ -164,6 +164,12 @@ plan.addSelection("BackupPlanSelection", {
 
 For example, if you would like to retain a resource on stack deletion, you can use the `applyRemovalPolicy` property on the resource to add a retention policy.
 
+<Callout>
+
+Retaining resources on stack deletion is not supported in sandbox.
```

Comment:
I'd reword this to something like
"`sandbox delete` ignores any resource removal policy and deletes all resources always."

Since the stack deletion outside of amplify would still respect these removal policy (e.g. using CDK or CFN directly)